sdk/nodejs/cloudfunctions/index.ts

```diff
@@ -4,3 +4,4 @@
 // Export members:
 export * from "./function";
 export * from "./getFunction";
+export * from "./zMixins";
```

Comment:
documented in resources.go.  We need the mixins to be loaded last, in order for modules to properly loaded so that we can update the prototype of existing classes.  our generator always sorts these alphabetically.   so this is a way of getting them at the end.  hacky.. but it works